docs/install-kafka-operator.md

```diff
@@ -32,24 +31,23 @@ This method uses a command-line tool of the commercial [Banzai Cloud Supertubes]
     supertubes install -a
     ```
 
-## Install Kafka operator and the requirements independently {#manual-install}
+## Install {{< kafka-operator >}} and the requirements independently {#manual-install}
 
 ### Install cert-manager {#install-cert-manager}
 
-The Kafka operator uses [cert-manager](https://cert-manager.io) for issuing certificates to clients and brokers. Deploy and configure cert-manager if you haven't already done so.
+{{< kafka-operator >}} uses [cert-manager](https://cert-manager.io) for issuing certificates to clients and brokers. Deploy and configure cert-manager if you haven't already done so.
 
 > Note:
->
-> - Kafka operator 0.8.x and newer supports cert-manager 0.15.x
-> - Kafka operator 0.7.x supports cert-manager 0.10.x
+> - {{< kafka-operator >}} 0.18.1 and newer supports cert-manager 1.5.3-1.6.x
```

Comment:
**Question**: @stoader can we declare support for cert-manager 1.8, what do you think?
IMO we should be clear based on the release notes and the depended library version, alhough I haven't tested that manually yet.

(If we change the version, the corresponding commands also need to be updated with the new version below.)

Reply:
I'm going to create a task for testing this and we are going to get back here with the result.

Reply:
Nevermind, if it comes from the internal koperator docs then let's have this updated there and have it be propagated here from there instead of changing it here.